Pipfile

```diff
@@ -4,31 +4,33 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-bandit = "*"
+bandit = "==1.6.2"
 nose = "*"
-pytest = "*"
-pylint = "*"
-autopep8 = "*"
+pytest = "==5.2.1"
+pylint = "==2.4.2"
+autopep8 = "==1.4.4"
 setuptools = "*"
-wheel = "*"
-schema = "*"
+wheel = "==0.33.6"
+schema = "==0.7.1"
+safety = "==1.8.5"
+invoke = "1.3.0"
```

Comment:
This should be `invoke = "==1.3.0"`

Reply:
I was getting an error from PyCharm without it; looks legit since it should prob follow the same format as the other ones.